internal/queue/tests/main_test.go

```diff
@@ -0,0 +1,41 @@
+//go:build integration
+// +build integration
+
+// To override application configuration for integration tests, create config/test.env file.
+package tests
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/RHEnVision/provisioning-backend/internal/config"
+	_ "github.com/RHEnVision/provisioning-backend/internal/dao/pgx"
+	_ "github.com/RHEnVision/provisioning-backend/internal/logging/testing"
+	"github.com/RHEnVision/provisioning-backend/internal/testing/integration"
+)
+
+// truncate and seed database tables
+func reset() {
+	integration.DbSeed()
+}
+
+func TestMain(t *testing.M) {
+	ctx := context.Background()
+	ctx = integration.InitConfigEnvironment(ctx, "../../../config/test.env")
+	// override the default value to reasonable timeout for testing
+	config.Worker.Queue = "redis"
+	config.Worker.Timeout = 1 * time.Second
```

Comment:
I'm really nitpicking here, pls feel free to ignore me, but this would look nice in a function (`configureJobqueue` for example).
But I'm all for merging as is and addressing this in a follow-up, if I get around to this, such a global setting would look nice in a config file, that we would create for the GH actions (definitelly out of scope this PR).

Reply:
Don’t exactly understand what you mean.